src/Components/App/App.js

```diff
@@ -22,13 +21,25 @@ function App() {
         }
       })
     }, [])
+    
+  const updateSelectedMovie = (event) => {
```

Comment:
Putting updateSelected movie here is a good call, we have to pass down less props and are not passing down the setter function.